.github/workflows/release.yml

```diff
@@ -60,23 +60,26 @@ jobs:
     environment: release
     runs-on: ubuntu-22.04
     needs: pypi
+    if: github.event_name == 'release'
```

Comment:
Avoid running the `bump-homebrew-formula` job if the event was workflow_dispatch, since the `github.event.release.tag_name` value will not be present.